tests/foreman/destructive/conftest.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from fauxfactory import gen_string
+from requests.exceptions import HTTPError
+
+from robottelo.constants import DEFAULT_ORG
+from robottelo.logging import logger
+
+
+@pytest.fixture
+def default_org(target_sat):
+    return target_sat.api.Organization().search(query={'search': f'name="{DEFAULT_ORG}"'})[0]
```

Comment:
1. default org in global fixture uses the session scope and that would cause an adverse effects in destructive tests .

2. I tried to move the ui conftest module user fixture globally but there are some conflicts with module user exists in the component fixtures as well. It is taking the precedence over the pytest_fixtures/component module_user fixture than the pytest_fixtures/core module_user fixture

Reply:
I can modify the fixture to ui_user. that will help to find it uniquely and do not conflict

Reply:
I think it's appropriate to have fixtures defined in conftest here due to the scope of the directory and the fixture scopes applied to the global fixtures.